mmv1/products/cloudrun/api.yaml

```diff
@@ -616,6 +616,79 @@ objects:
+                  description: |-
+                    Number of seconds after the container has started before the probe is
+                    initiated.
+                    Defaults to 0 seconds. Minimum value is 0. Maximum value is 240.
+                  default_value: 0
+                - !ruby/object:Api::Type::Integer
+                  name: timeoutSeconds 
+                  description: |-
+                    Number of seconds after which the probe times out.
+                    Defaults to 1 second. Minimum value is 1. Maximum value is 3600.
+                    Must be smaller than periodSeconds.
+                  default_value: 1
+                - !ruby/object:Api::Type::Integer
+                  name: periodSeconds 
+                  description: |-
+                    How often (in seconds) to perform the probe.
+                    Default to 10 seconds. Minimum value is 1. Maximum value is 240.
+                  default_value: 10
+                - !ruby/object:Api::Type::Integer
+                  name: failureThreshold  
+                  description: |-
+                    Minimum consecutive failures for the probe to be considered failed after
+                    having succeeded. Defaults to 3. Minimum value is 1.
+                  default_value: 3
+                - !ruby/object:Api::Type::NestedObject
+                  name: tcpSocket 
+                  description: |-
+                    TcpSocket speficies an action involving a TCP port.
+                  send_empty_value: true
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: port
+                      description: |-
+                        Port number to access on the container. Number must be in the range 1 to 65535.
+                - !ruby/object:Api::Type::NestedObject
+                  name: httpGet 
+                  description: |-
+                    HttpGet specifies the http request to perform.
+                  send_empty_value: true
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: path
+                      description: |-
+                        Path to access on the HTTP server. If set, it should not be empty string.
+                      default_value: "/"
+                    - !ruby/object:Api::Type::Array
+                      name: httpHeaders
+                      description: |-
+                        Custom headers to set in the request. HTTP allows repeated headers.
+                      item_type: !ruby/object:Api::Type::NestedObject
+                        properties:
+                          - !ruby/object:Api::Type::String
+                            name: name
+                            description: |-
+                              The header field name.
+                            default_value: ""
+                            send_empty_value: true
```

Comment:
What does empty mean here? What I want to understand is why a user would want to specify empty rather than some value. Sending an empty header field doesn't make sense, so does empty have a special meaning to the API?

Reply:
You're right, empty header name doesn't make sense. I checked what GKE does and GKE doesn't allow empty. Our API needs a change.

Reply:
Should we wait on that change before releasing this then?

Reply:
I don't think we need to wait since in Terraform side this field is required.

For header value, IIUC, empty is a valid http header value. Just verified GKE also supports empty value.